event-processing/event-processor/services/validation-service.ts

```diff
@@ -96,4 +92,8 @@ export class ValidationService {
 
         return unknownFields;
     }
+
+    private static isDate = (timestamp: number): boolean => {
```

Comment:
Is there a reason we're checking >0 rather than before current time or anything like that? (I'm presuming there is)

Reply:
It's just a simple date validation so >0 proves its a valid timestamp. No requirement for anything more complex.

Reply:
Great, just checking